backends-velox/src/main/scala/org/apache/gluten/backendsapi/velox/VeloxRuleApi.scala

```diff
@@ -75,6 +75,7 @@ private object VeloxRuleApi {
     injector.injectTransform(_ => CollapseProjectExecTransformer)
     injector.injectTransform(c => FlushableHashAggregateRule.apply(c.session))
     injector.injectTransform(c => InsertTransitions(c.outputsColumnar))
+    injector.injectTransform(c => PartialProjectRule.apply(c.session))
```

Comment:
minor: Can we move this in between `TransformPreOverrides` and `RemoveNativeWriteFilesSortAndProject`? Thanks!